client/src/test/resources/AssociativeArrayOfLists.yaml

```diff
@@ -1,6 +0,0 @@
-# Block associated array of lists (showing examples of both inline and blocked lists)
```

Comment:
Yes this removal was accidental.  The files should have been moved into dataprovider module.